assets/src/blocks/SecondaryNavigation/SecondaryNavigationFrontend.js

```diff
@@ -0,0 +1,330 @@
+import {useState, useEffect, useRef} from '@wordpress/element';
+import {getHeadingsFromDom} from '../TableOfContents/getHeadingsFromDom';
```

Comment:
It's confusing to use a function from the `TableOfContents` folder here, can we move this function somewhere else instead? Maybe in `src/functions` folder?